app/experimenter/experiments/constants/nimbus.py

```diff
@@ -409,6 +409,30 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_INFREQUENT_USER_NEED_DEFAULT_HAS_PIN = NimbusTargetingConfig(
+    name="Infrequent user (need default, has pin)",
+    slug="infrequent_user_need_default_has_pin",
+    description="Between 1 & 6 days activity in past 28 days need default w/ pin",
+    targeting=("{infrequent_need_default} && !doesAppNeedPin || {sticky}").format(
```

Comment:
How about factoring out a separate targeting which just aliases "!doesAppNeedPin" to TARGETING_HAS_PIN and using that?

Reply:
That should make things more readable.